tests/software/gatk4/haplotypecaller/main.nf

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { GATK4_HAPLOTYPECALLER } from '../../../../software/gatk4/haplotypecaller/main.nf' addParams( options: [:] )
+
+workflow test_gatk4_haplotypecaller {
+    
+    def input = []
```

Comment:
```suggestion
```

Reply:
Don't really need this. Will get it removed from the modules template too at some point.